universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -32,6 +32,18 @@
 import services.applicant.predicate.JsonPathPredicate;
 import services.applicant.question.Scalar;
 
+/**
+ * Brokers access to the answer data for a specific applicant at a specific version.
```

Comment:
Thought (non-blocking): I might be wrong but I thought sharing applicant data across versions is how we share answers across versions. `at a specific version` sounds odd to me. Would love to be convinced otherwise!

Reply:
removed `at a specific version` in light of the discussion at standup today